cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/SteadyStateHypothesisExport.java

```diff
@@ -28,93 +28,93 @@
  * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
  * @author Luma Zamarreño <zamarrenolm at aia.es>
  */
-public final class SteadyStateHypothesisExport {
+public final class SteadyStateHypothesisExport extends AbstractCgmesExporter {
 
     private static final Logger LOG = LoggerFactory.getLogger(SteadyStateHypothesisExport.class);
     private static final String REGULATING_CONTROL_PROPERTY = Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + "RegulatingControl";
     private static final String GENERATING_UNIT_PROPERTY = Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + "GeneratingUnit";
 
-    private SteadyStateHypothesisExport() {
-    }
+    private final Map<String, List<RegulatingControlView>> regulatingControlViews = new HashMap<>();
 
-    public static void write(Network network, XMLStreamWriter writer, CgmesExportContext context) {
-        final Map<String, List<RegulatingControlView>> regulatingControlViews = new HashMap<>();
-        String cimNamespace = context.getCim().getNamespace();
+    SteadyStateHypothesisExport(CgmesExportContext context, XMLStreamWriter xmlWriter) {
+        super(context, xmlWriter);
+    }
 
+    public void export() {
```

Comment:
You can add an abstract `export()` method in `AbstractCgmesExporter` (to be implemented in inherited classes)

Reply:
Didn't realize it was already the case! Only add `@Override` to indicate it.

Reply:
Added the missing `@override` annotations.